functions/tips-scopes/main.py

```diff
@@ -15,7 +15,11 @@
 
 import time
 
+import functions_framework
```

Comment:
Sorry I wasen't clear before, I was typing in a hurry; I think this looks good, except I would put the `import functions_framework` and `import time` inside the region tags too, so that the snippet is a bit more copy-pastable.